Document/0x02b-MASVS-MASTG-Adoption.md

```diff
@@ -65,7 +65,7 @@ In 2021, ioXt has [extended its security principles through the Mobile Applicati
 
 | Name | Document | Year |
 | -------------- | -------------------- | -- |
-| Government of Singapore, Cyber Security Agency (CSA) | [Safe App Standard](https://www.csa.gov.sg/docs/default-source/publications/2024/safe-app-standard-(v1-0)a4e34e8d-2679-44d1-a836-3904922d7b6a.pdf?sfvrsn=8732135_1) | 2024 |
+| Government of Singapore, Cyber Security Agency (CSA) | [Safe App Standard v2.0](https://isomer-user-content.by.gov.sg/36/b4442074-5f44-40ad-a1b5-1b45965362c3/safe-app-standard-2-0.pdf) | 2024 |
```

Comment:
```suggestion
| Government of Singapore, Cyber Security Agency (CSA) | [Safe App Standard v2.0](https://www.csa.gov.sg/resources/publications/safe-app-standard-2-0) | 2024 |
```

Let's use the main landing page, this page has links to the PDF, the online version and an infographic.